apps/src/templates/teacherDashboard/teacherSectionsRedux.js

```diff
@@ -589,10 +591,22 @@ export default function teacherSections(state=initialState, action) {
         throw new Error(`Cannot edit property ${key}; it's not allowed.`);
       }
     }
+
+    // Selecting Course 1-4 or A-F should auto-enable Stage Extras.
+    if (action.props.scriptId) {
+      const script = state.validAssignments[`null_${action.props.scriptId}`];
+      if (script && /course[1-4a-f]/.test(script.script_name)) {
```

Comment:
Would be good to make `has_lesson_extras` a property on script.rb, but I think this is ok for now.